components/measurement/nettests/WebConnectivity.js

```diff
@@ -338,13 +330,12 @@ const WebConnectivityDetails = ({
           WebsiteURL: input,
           network: probe_asn,
           country: country,
-          BlockingReason: <strong>{messages[`blockingReason.${blocking}`] && intl.formatMessage(messages[`blockingReason.${blocking}`])}</strong>
+          BlockingReason: reason && <strong>{reason}</strong>
```

Comment:
Maybe we don't even have to do this `&&`, because worse case we create an empty `<strong></strong>` element.

Reply:
I generally try not to let unnecessary elements into the DOM. It has led layout issues in the past, although not exactly with this type of an element.

Reply:
That said, I don't claim that none have slipped past me :grin: